utils/zms-cli/zms-cli.go

```diff
@@ -154,6 +154,7 @@ func usage() string {
 	buf.WriteString("   -k                  Disable peer verification of SSL certificates.\n")
 	buf.WriteString("   -key x509_key       Athenz X.509 Key file for authentication\n")
 	buf.WriteString("   -o output_format    Output format - json or yaml (default=yaml)\n")
+	buf.WriteString("   -overwrite           Overwrites without checking for existence\n")
```

Comment:
not to be super picky but there is an extra space before Overwrites so let's make sure our online help is all aligned

Reply:
thank you. ill check it